copilot/post-award/manifest.yml

```diff
@@ -114,8 +114,8 @@ environments:
           CLIENT_MAX_BODY_SIZE: 10m
           PROXY_READ_TIMEOUT: 180s
         secrets:
-          BASIC_AUTH_USERNAME: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/POST_AWARD_BASIC_AUTH_USERNAME
-          BASIC_AUTH_PASSWORD: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/POST_AWARD_BASIC_AUTH_PASSWORD
```

Comment:
Can we manually delete these once it is deployed? As they won't get removed automatically.

Reply:
Good shout, It is Done!